Basic structure

mikethreels opened the pull request:
## Milestone: Basic file structure 

### When updating the project the following points have been added.
- [x] Initialized app with create-react-app
- [x] Deleted all unnecessary files or assets.
- [x] Added prop-types using npm.
- [x] Added Redux packages
- [x] Deployed to Heroku.

mikethreels committed:
feat: replace yarn.lock with package-lock.json

mikethreels committed:
docs: deleted unnecessary files

mikethreels committed:
feat: installed prop-types package

mikethreels committed:
feat: installed big.js using npm

mikethreels committed:
docs: deleted remaining unnecessary files

mikethreels committed:
feat: setup linters

mikethreels committed:
fix: fixed linter errors

mikethreels committed:
feat: created template for readme file

mikethreels committed:
fix: fixed linter errors

Torres-ssf commented on src/index.js:
```diff
@@ -1,17 +1,2 @@
-// to log results (for example: reportWebVitals(console.log))
-// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
-reportWebVitals();
+// eslint-disable-next-line no-console
+console.log('hello world');
```
- [x] You did a good job deleting all unnecessary files. However, we need to let `src/index.js` file to remain connected with React.

```JS
import React from 'react';
import ReactDOM from 'react-dom';

ReactDOM.render(
  <React.StrictMode>
    <h1>Hello World</h1>
  </React.StrictMode>,
  document.getElementById('root'),
);

```

Torres-ssf reviewed (changes requested):
### Changes required :muscle: 
Hi @mikethreels :hand: 
Awesome job on this milestone so far. but we have to make some changes before moving on.

- [x] The live link on your README file it's not working right now, We need to make sure that your app is pushed to Heroku or Netlify.
![Screenshot from 2020-11-12 19-36-16](https://user-images.githubusercontent.com/30977845/99004743-5cbbf300-251e-11eb-8ac7-ab96cffc47e4.png)

Feel free to message me on slack if you have any questions :smile: 
Happy Coding :computer: 
**Sergio Torres**

mikethreels committed:
feat: added live link to readme file

mikethreels committed:
feat: made adjustments as requested by the TSE

mikethreels commented:
## Milestone: Basic file structure 

### When updating the project the following points have been added.
- [x] Added live link to the readme file
- [x] Added import from React as requested by the TSE

lucasmazo23 reviewed (approved):
## Project approved 🚀
### Great job!
### Your project looks great!
### You can go ahead and merge this pull request!
### Happy coding! 💻
## Lucas Mazo
